packages/core/core-flows/src/definition/cart/steps/validate-cart-shipping-options.ts

```diff
@@ -5,6 +5,10 @@ import { createStep, StepResponse } from "@medusajs/workflows-sdk"
 
 interface StepInput {
   cart: CartDTO
+  shippingOptionsContext: {
```

Comment:
For now these are the only things we can set through the UI, we can revisit this check once we get closer to prod readiness